### ISSUE TITLE
Kind inquiry about the dataset VGGFace2

### ISSUE THREAD

**Issue by pxzhang94** [seymayucer/VGG1200-Races#1]
the link of VGGFace 2 (https://www.robots.ox.ac.uk/~vgg/data/vgg_face2/) is not available, is it possible there is another link we can access

**Comment by seymayucer** [comment]
hello, you can download many datasets from here: https://github.com/ZhaoJ9014/face.evoLVe.PyTorch, for 112x112 aligned images of VGGFace2 you can use this link: https://drive.google.com/file/d/1N7QEEQZPJ2s5Hs34urjseFwIoPVSmn4r/view

**Comment by pxzhang94** [comment]
Hi Seyma,
Thanks for your kind reply, it is a very useful repo for my research. However, I'm studying on the fairness of DL models, I wonder if it's possible to get the dataset VGGFace2 1200.
Thanks in advance! Have a good day!


Zhang Peixin

Ph.D Candidate

College of&nbsp;Computer Science and Technology, Zhejiang University

+86 18868106685





&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "seymayucer/VGG1200-Races"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年3月22日(星期一) 晚上6:26
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [seymayucer/VGG1200-Races] Kind inquiry about the dataset VGGFace2 (#1)





 
hello, you can download many datasets from here: https://github.com/ZhaoJ9014/face.evoLVe.PyTorch, for 112x112 aligned images of VGGFace2 you can use this link: https://drive.google.com/file/d/1N7QEEQZPJ2s5Hs34urjseFwIoPVSmn4r/view

**Comment by seymayucer** [comment]
Hello, you can simply take images from the datasets by using VGGFace2_1200_Train_Subjects_Metadata.csv.But if you are asking for vggface2-1200-races then you can generate images using the pre-trained models.

**Comment by pxzhang94** [comment]
Sorry to bother you again, is it possible to share the trained face recognition models (ResNet 100) with batchnorm-dropout-fc-batchnorm-softmax&nbsp;before and after applying the race&nbsp;transfer?




Zhang Peixin

College of&nbsp;Computer Science and Technology, Zhejiang University





&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "seymayucer/VGG1200-Races"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年3月23日(星期二) 凌晨3:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [seymayucer/VGG1200-Races] Kind inquiry about the dataset VGGFace2 (#1)





 
Hello, you can simply take images from the datasets by using VGGFace2_1200_Train_Subjects_Metadata.csv.But if you are asking for vggface2-1200-races then you can generate images using the pre-trained models.
